README.md

```diff
@@ -34,54 +34,56 @@ pip install -r requirements.txt
 
 ### 3. (Optional) Start a local server for testing
 
-You can start a local server by running `python -m server` if you want to test the client locally. This will start a server on `0.0.0.0:5555`.
+You can start a local server by running `python -m server` if you want to test the client locally. This will start a server on `0.0.0.0:5555` (the host set in the configuration file).
 Then, open another terminal, go to the project folder, and run `python -m client config.json` to connect to the local server. This is optional, but recommended for testing before connecting to the remote server.
 
 This allows:
-    - You to connect locally with your own client
-    - Other players to connect to your game if you share your IP address with them
-    - This is useful for organizing your own competitions or testing with friends
+- You to connect locally with your own client
+- Other players to connect to your game if you share your IP address with them
+- This is useful for organizing your own competitions or testing with friends
 
 ### 4. Select a Game Mode
 
 The game supports three different modes that can be set in the `config.json` file:
 
-
-- **Agent Mode** (`"game_mode": "agent"`): In this mode, the client connects to a remote server to compete against other players' agents in a battle. The client uses the agent specified in the `agent` field of the configuration file.
+- **Agent Mode** (`"game_mode": "agent"`): In this mode, the client connects to a remote server to compete against other players' agents. The client uses the agent specified in the `agent` field of the configuration file.
   
-- **Manual Mode** (`"game_mode": "manual"`): In this mode, the client connects to a remote server to compete against other players' agents in a battle. The client does not use an agent and instead controls the train manually with keyboard arrows.
+- **Manual Mode** (`"game_mode": "manual"`): In this mode, the client connects to a remote server to compete against other players' agents. The client does not use an agent and instead controls the train manually with keyboard arrows.
 
-- **Observer Mode** (`"game_mode": "observer"`): This mode will make your client an observer. If no other client connects, the agents specified in the `local_agents` list in `config.json` will compete against each other, allowing you to test and compare different versions of your agents. For better organization, it's recommended to store your agents in the "agents" folder.
+- **Observer Mode** (`"game_mode": "observer"`): This mode will make your client an observer and you can only watch the game without interacting. If no other client connects, random agents from the `agents` list in `config.json` will compete against each other, allowing you to test and compare different versions of your agents. For better organization, it's recommended to store your agents in the "agents" folder.
```

Comment:
~it's recommended ...~ => you must store your agents in the common/agents/ folder.